web/app/js/components/EmptyCard.jsx

```diff
@@ -12,12 +13,12 @@ const styles = () => ({
   },
 });
 
-const EmptyCard = ({ content, classes }) => {
+const EmptyCard = ({ content, classes, t }) => {
   return (
     <Card className={classes.card} elevation={3}>
       <CardContent>
         <Typography>
-          {content}
+          {t(content)}
```

Comment:
IMO I think it's better to always try calling `t` on string literals when possible, so in this case below in `EmtpyCard.defaultProps` and in `EmptyCard`'s calling sites, so as to allow automated tools to pull all the strings and complete the json translation files.

Reply:
I can't actually find any examples in the codebase where we use an empty card with non-default text? Feels like we could just simplify this component up until the extra functionality is needed. 

That said, I'm +1, the translated content should be passed down pre-translated as part of the prop.